.builds/openbsd.yml

```diff
@@ -31,3 +31,8 @@ tasks:
     cmake -G Ninja ..
     cmake --build . --config Debug
     ./bin/nvim --version
+- test: |
+    export LC_CTYPE=en_US.UTF-8
+    cd neovim
```

Comment:
Maybe "could not load cache" is fixed by:

    cd neovim/build

Reply:
Alas no.

I don't think the old tests are CMakeified:
```
$ pwd             
/home/edd/source/neovim
$ find . -name 'CMake*' | xargs grep testdir 
$ 
```

Even the appveyor test you referenced earlier uses `make` for old tests:
https://github.com/neovim/neovim/blob/e2ccf47b5e5e5acf19a74d928e367c7726d29ad3/ci/build.ps1#L137-L143

Reply:
By the way, does `VERBOSE=1` actually do anything above? I tried adding this to my invocation and don't see any additional output.